README.md

```diff
@@ -0,0 +1,85 @@
+| openvpn_server_netmask             | `255.255.255.0`                                   | Netmask of the private network                                                                                                                                  |
+| openvpn_tls_auth_required          | `true`                                            | Ask the client to push the generated ta.key of the server during the   connection                                                                               |
+| openvpn_server_ipv6_network        | `false`                                           | If set, the network address and prefix of an IPv6 network to assign to   clients. If True, IPv4 still used too.                                                 |
+| openvpn_ca_key                     |                                                   | Contain "crt" and "key". If not set, CA cert and key   will be automatically generated on the target system.                                                    |
+| openvpn_tls_auth_key               |                                                   | Single item with a pre-generated TLS authentication key.                                                                                                        |
+| openvpn_topology                   | `false`                                           | the "topology" keyword will be set in the server config with   the specified value.                                                                             |
+| openvpn_push                       | `empty`                                           | Set here a list of string that will be placed as "push   "<string>". E.g `- route 10.20.30.0 255.255.255.0` will generate push "route 10.20.30.0 255.255.255.0" |
+| openvpn_use_ldap                   | `false`                                           | Active LDAP backend for authentication. Client certificate not needed   anymore                                                                                 |
+| openvpn_ldap                       |                                                   | Dictionary that contain LDAP configuration                                                                                                                      |
+| openvpn_crl_path                   |                                                   | Define a path to the CRL file for revokations.                                                                                                                  |
+| openvpn_use_crl                    |                                                   | Configure OpenVPN server to honor certificate revocation list.                                                                                                  |
```

Comment:
Again. Why don't you just use `when: openvpn_crl_path is defined`?

Reply:
because there is a default path that fits into the directory structure only if you want your own path you should change crl_path